README.md

```diff
@@ -1,27 +1,81 @@
-# HMPPS Prisoner Search Indexer
+# HMPPS Prisoner Search
 
-[![repo standards badge](https://img.shields.io/badge/dynamic/json?color=blue&style=flat&logo=github&label=MoJ%20Compliant&query=%24.result&url=https%3A%2F%2Foperations-engineering-reports.cloud-platform.service.justice.gov.uk%2Fapi%2Fv1%2Fcompliant_public_repositories%2Fhmpps-prisoner-search-indexer)](https://operations-engineering-reports.cloud-platform.service.justice.gov.uk/public-github-repositories.html#hmpps-prisoner-search-indexer "Link to report")
-[![CircleCI](https://circleci.com/gh/ministryofjustice/hmpps-prisoner-search-indexer/tree/main.svg?style=svg)](https://circleci.com/gh/ministryofjustice/hmpps-prisoner-search-indexer)
-[![Docker Repository on Quay](https://quay.io/repository/hmpps/hmpps-prisoner-search-indexer/status "Docker Repository on Quay")](https://quay.io/repository/hmpps/hmpps-prisoner-search-indexer)
-[![API docs](https://img.shields.io/badge/API_docs_-view-85EA2D.svg?logo=swagger)](https://prisoner-search-indexer-dev.prison.service.justice.gov.uk/swagger-ui/index.html?configUrl=/v3/api-docs)
+[![CircleCI](https://circleci.com/gh/ministryofjustice/hmpps-prisoner-search/tree/main.svg?style=svg)](https://circleci.com/gh/ministryofjustice/hmpps-prisoner-search)
+[![Docker](https://quay.io/repository/hmpps/hmpps-prisoner-search/status)](https://quay.io/repository/hmpps/hmpps-prisoner-search/status)
+[![API docs](https://img.shields.io/badge/API_docs_-view-85EA2D.svg?logo=swagger)](https://prisoner-search-dev.prison.service.justice.gov.uk/swagger-ui/index.html?configUrl=/v3/api-docs/swagger-config)
+
+This README provides information on the prisoner search service.  For the indexer service that keeps the search
+up-to-date from Prison systems (NOMIS) please see the [indexer README](hmpps-prisoner-search-indexer/README.md).
```

Comment:
this README is now about the search - with separate readme for indexer as most people will only be interested in the search API